packages/nimble-components/src/table/tests/table.stories.ts

```diff
@@ -48,10 +49,22 @@ const dataSetIdFieldNames = {
 
 const overviewText = 'The `nimble-table` is a component that offers a way to render tabular data in a variety of ways in each column.';
 
-const dataDescription = `\`data\` is a property that is an array of records. A record provides the data that backs a single row in the table.
-Each record is made up of fields, which are key/value pairs. The key in each pair must be of type \`string\`, which is defined by the type
-\`TableFieldName\`. The value in each pair must be of type \`string\`, \`number\`, \`boolean\`, \`Date\`, \`null\`, or \`undefined\`,
-which is defined by the type \`TableFieldValue\`.`;
+const dataDescription = `To set the data on the table, call \`setData()\` with an array data records. Each record is made up of fields,
+which are key/value pairs. The key in each pair must be of type \`string\`, which is defined by the type \`TableFieldName\`. The value
+in each pair must be of type \`string\`, \`number\`, \`boolean\`, \`Date\`, \`null\`, or \`undefined\`, which is defined by the type \`TableFieldValue\`.
+
+The table will not automatically update if the contents of the array change after calling \`setData()\`. To trigger an update, call
+\`setData()\` again with the same array reference or with a new array.
+
+<details>
+    <summary>Framework specific considerations</summary>
+    - Angular: In addition to exposing the \`setData()\` function in Angular, you can use the \`data$\` property to provide an
+    \`Observable<TableRecord[]>\`. Nimble will automatically subscribe and unsubscribe to the provided \`Observable\` and call
+    \`setData()\` on the web component when new values are emitted.
+    - Blazor: In addition to exposing the \`setData\` function in Blazor, you can also use the \`Data\` property on the
```

Comment:
This isn't accurate right? We aren't exposing the `setData` function in Blazor at all.

Reply:
Yes, you're right. Good catch!